movie_director_insert.sql

```diff
@@ -0,0 +1,574 @@
+use movie;
```

Comment:
These .sql files shouldn't have been committed, gotta keep github free of clutter.